querybook/webapp/components/DataTableViewOverview/DataTableViewOverview.tsx

```diff
@@ -130,6 +130,16 @@ export const DataTableViewOverview: React.FC<
         />
     ) : null;
 
+    const tableLinks = table.table_links ?? [];
+    const tableLinksDOM = tableLinks.map((link, index) => (
```

Comment:
nit: seems unnecessary to introduce a new variable, just `(table.table_links ?? []).map`